src/sys/event.rs

```diff
@@ -96,6 +96,27 @@ pub enum EventFilter {
     EVFILT_TIMER = -7,
     EVFILT_EXCEPT = -8,
     EVFILT_USER = -9,
+    EVFILT_FS = -10,
+    EVFILT_SYSCOUNT = 10,
+}
+
+#[cfg(target_os = "freebsd")]
+#[repr(i16)] // u_short
+#[derive(Clone, Copy, Debug, PartialEq)]
+pub enum EventFilter {
+    EVFILT_READ = -1,
```

Comment:
Can we use the constants from libc instead (adding as needed).  They have better tests for such things there and it is the way we have been moving things.